release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -291,6 +291,34 @@ Currently, it is not supported to use Active Directory Federation Services with
 
 For the full list of OIDC providers, see xref:../authentication/identity_providers/configuring-oidc-identity-provider.adoc#identity-provider-oidc-supported_configuring-oidc-identity-provider[Supported OIDC providers].
 
+[id="ocp-4-11-auth-pod-security-admission"]
+==== Pod security admission autolabeling
+
+link:https://kubernetes.io/docs/concepts/security/pod-security-admission/[Pod security admission] is now enabled on {product-title}. 
+
+If the pod security admission plug-in is activated, {product-title} preserves pod workloads despite changes to the pod security API. A cluster performs this action by monitoring administrator-labeled namespaces for changes and mapping link:https://kubernetes.io/docs/concepts/security/pod-security-standards/[pod security standards] to xref:../authentication/managing-security-context-constraints.adoc[security context constraints].
+
+You can learn more about admission controller configuration in the link:https://kubernetes.io/docs/tasks/configure-pod-container/enforce-standards-admission-controller/#configure-the-admission-controller[Kubernetes documentation].
+
+The `privileged` pod security profile is enforced, and the `restricted` profile is used for warnings and audits.
+
+This release also introduces new security context constraints (SCCs) that enforce pod security in accordance with new pod security admission controls. These SCCs are:
```

Comment:
I think we would like to mention that this change was to improve the workloads security to be able to reach the "restricted" pod security profile, something like:

`In order to improve workloads security according to the "restricted" pod security profile, this release also introduces..`

Reply:
```suggestion
To improve workload security by using the "restricted" pod security profile, this release introduces security context constraints (SCCs) that enforce pod security in accordance with new pod security admission controls. These SCCs are:
```